beacon_node/http_api/tests/interactive_tests.rs

```diff
@@ -940,3 +940,106 @@ async fn queue_attestations_from_http() {
 
     attestation_future.await.unwrap();
 }
+
+// Test that a request for next epoch proposer duties suceeds when the current slot clock is within
+// gossip clock disparity (500ms) of the new epoch.
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn proposer_duties_with_gossip_tolerance() {
+    let validator_count = 24;
+    let spec = ForkName::Fulu.make_genesis_spec(E::default_spec());
+
+    let tester = InteractiveTester::<E>::new(Some(spec.clone()), validator_count).await;
```

Comment:
we run all http-api tests on electra and fulu, so this test will get run twice with the same Fulu genesis conifg.Should we just pass `None` so we test two different forks?